src/domains/message/pages/VerifyMessage/VerifyMessage.tsx

```diff
@@ -68,6 +136,18 @@ export const VerifyMessage: React.VFC = () => {
 		}
 	};
 
+	const isSubmitDisabled = () => {
+		if (isSubmitting) {
+			return true;
+		}
+
+		if (isDirty || Object.values(initialState).every(Boolean)) {
+			return !isValid;
```

Comment:
@dated seems that RHF is giving `isValid` false when switching from manual form to json view using the toggle, resulting into having the "Verify" button disabled. It's not very clear why it invalidates the state but we might as well check against form errors to see if they are empty. :hmm:

```suggestion
			return Object.values(form.errors).length > 0;

```